src/cgi/execve.cpp

```diff
@@ -6,6 +6,17 @@
 
 #define BUFFER_SIZE 4092
 
+std::string
+get_query(const std::map<std::string, std::string> &map)
+{
+	std::map<std::string, std::string>::const_iterator it = map.find("Query");
+	std::string										   ret;
+	ret = "";
+	if (it != map.end() && !it->second.empty())
+		ret = it->second;
+	return (ret);
```

Comment:
That avoids 
1. default constructor
2. equal operator
```suggestion
	std::map<std::string, std::string>::const_iterator it = map.find("Query");
	if (it != map.end() && !it->second.empty())
		return it->second;
	return "";
```